feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/steps/Step.kt

```diff
@@ -91,13 +94,14 @@ abstract class StepResultMixin : StepResult
         name = "EnrolLastBiometricStepResult.CaptureResult",
     ),
     JsonSubTypes.Type(value = ExternalCredentialParams::class, name = "ExternalCredentialParams"),
-    JsonSubTypes.Type(value = ExternalCredentialSearchResult::class, name = "ExternalCredentialSearchResult"),
-    JsonSubTypes.Type(value = MatchResult::class, name = "MatchResult"),
     // Additional types that are used in top-level params
     JsonSubTypes.Type(value = CaptureSample::class, name = "CaptureSample"),
     JsonSubTypes.Type(value = MatchConfidence::class, name = "MatchConfidence"),
     JsonSubTypes.Type(value = BiometricDataSource::class, name = "BiometricDataSource"),
+    JsonSubTypes.Type(value = BiometricDataSource.CommCare::class, name = "BiometricDataSource.CommCare"),
+    JsonSubTypes.Type(value = BiometricDataSource.Simprints::class, name = "BiometricDataSource.Simprints"),
     JsonSubTypes.Type(value = SubjectQuery::class, name = "SubjectQuery"),
+    JsonSubTypes.Type(value = AgeGroup::class, name = "AgeGroup"),
```

Comment:
Hm, changes in this file seem suspicious - were the added ones missed previously? And the deleted ones unneeded?